gdal/configure.ac

```
@@ -1295,6 +1317,8 @@ else
     fi
     if test "$PROJ_FOUND" = "no"; then
         AC_MSG_ERROR([PROJ 6 symbols not found])
+    elif test x$with_proj != x/usr -a x$with_proj != x/ ; then
```

At least for consistence with the rest of the code:
```suggestion
    elif test "x$with_proj" != "x/usr" -a "x$with_proj" != "x/" ; then
```